main_sm:
  ros__parameters:
    start_pose: 
      position:
        x: -1.4325715422177885
        y: -4.693432258211892
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.6258864727722122
        w: 0.7799141768173976

    end_pose:
      position:
        x: 6.813752279850104
        y: -0.3935899702618913
        z: 0.0
      orientation:
        x: 0.0
        y: 0.0
        z: -0.10311866789382369
        w: 0.9946690607090397





